pyccl/_core/parameters/fftlog_params.py

```diff
@@ -52,7 +52,7 @@ def __repr__(self):
     def __eq__(self, other):
         if self is other:
             True
-        if type(self) != type(other):
+        if type(self) is not type(other):
```

Comment:
Code change. flake8 flagged this as more appropriate. It does the same thing as before.